test/baseline/completioninfo.spec.ts

```diff
@@ -0,0 +1,207 @@
+import Path from 'path'
+import { findPositionOrThrowIn, toNormalizedPath } from 'test/support/helpers'
+import { TestServer } from 'test/support/TestServer'
+
+describe('completioninfo', () => {
+  const projectPath = toNormalizedPath(
+    Path.resolve(__dirname, '../../samples/feature-completions'),
+  )
+
+  function abs(fileName: string) {
+    return toNormalizedPath(Path.resolve(projectPath, fileName))
+  }
+
+  let server: TestServer
+  beforeAll(async () => {
+    server = new TestServer()
+
+    await server.sendCommand('configure', { preferences: {}, watchOptions: {} })
+    await server.sendCommand('compilerOptionsForInferredProjects', {
+      options: {
+        allowJs: true,
+        checkJs: true,
+        allowNonTsExtensions: true,
+        jsx: 'preserve' as any,
+      },
+    })
+  })
+  afterEach(async () => await server.flush())
+  afterAll(async () => await server.close())
+
+  describe.each([
+    'Javascript.vue',
+    'JavascriptSetup.vue',
+    'Typescript.vue',
+    'TypescriptSetup.vue',
+    'Javascript.jsx',
+    'Typescript.tsx',
+  ])('in %s', (source) => {
+    const file = abs(`src/${source}`)
+
+    beforeAll(async () => {
+      await server.sendCommand('updateOpen', {
+        openFiles: [{ file, projectRootPath: projectPath }],
+      })
+    })
+
+    it('should provide info on import vue', async () => {
+      const { body } = await server.sendCommand(
+        'completionInfo',
+        await findPositionOrThrowIn(file, `import {} from 'vue'`, 8),
+      )
+
+      expect(body?.entries.length).toBeGreaterThan(1)
+      expect(body?.entries).toContainEqual({
```

Comment:
this actually returns +200 entries! 

I reckon the typescript language server will then filter them, but not sure

Reply:
We can look into this later, in current state it's a great addition to feature set.

Reply:
It works on the actual editor, on the logs it also runs the command: `getCompletionEntryDetails`